### PR TITLE
Eplb

### PR DESCRIPTION
**Pull request by qmkakaxi**
## Checklist

- [ ] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [ ] Please feel free to join our Slack channel at https://slack.sglang.ai to discuss your PR.

### CODE REVIEW

File: python/sglang/srt/layers/quantization/w8a8_int8.py [hunk 1]
```diff
@@ -955,12 +956,14 @@ def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
         layer.w13_weight = Parameter(
             layer.w13_weight.data.transpose(1, 2).contiguous(), requires_grad=False
         )
-        layer.w13_weight.data = torch_npu.npu_format_cast(layer.w13_weight.data, 29)
+        if global_server_args_dict["enable_ep_moe"]:
```
Comment (xiaobaicxy) [inline python/sglang/srt/layers/quantization/w8a8_int8.py:959]: w13的转nz直接删掉
